9c697d609329ca10c7067a418bba1022/update/sys_rest_message_fn_4c913aa1975512109790fefce053af8e.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_rest_message_fn">
+    <sys_rest_message_fn action="INSERT_OR_UPDATE">
+        <authentication_type>inherit_from_parent</authentication_type>
+        <basic_auth_password/>
+        <basic_auth_profile/>
+        <basic_auth_user/>
+        <content/>
+        <function_name>Get Plant Diseases</function_name>
+        <http_method>get</http_method>
+        <lock>false</lock>
+        <oauth2_profile/>
+        <protocol_name/>
+        <qualified_name>Plant disease import [ Get Plant Diseases ]</qualified_name>
+        <rest_endpoint>https://perenual.com/api/pest-disease-list?key=sk-oq9f6710caae7048c7302</rest_endpoint>
```

Comment:
Also holds your API key

Reply:
Hi @SapphicFire. I have removed the API key and used a query parameter instead.